src/Shared/Microsoft.AspNetCore.Razor.Serialization.Json/Microsoft.AspNetCore.Razor.Serialization.Json.projitems

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MSBuildAllProjects Condition="'$(MSBuildVersion)' == '' Or '$(MSBuildVersion)' &lt; '16.0'">$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    <HasSharedItems>true</HasSharedItems>
+    <SharedGUID>cd6913f3-ec47-4470-9c45-f5f898615e9d</SharedGUID>
+  </PropertyGroup>
+  <PropertyGroup Label="Configuration">
+    <Import_RootNamespace>Microsoft.AspNetCore.Razor.Serialization.Json</Import_RootNamespace>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="$(MSBuildThisFileDirectory)JsonDataConvert.cs" />
```

Comment:
Possibly, though this file is wholly controlled by VS tooling and doesn't allow editing without directly unloading the project. Given that, I would imagine that globbing isn't really a supported scenario for this file type. Also, while this is an MSBuild project file, it isn't an Sdk-style project. `.projitems` and `.shproj` are more akin to classic projects that the .NET SDK knows how to build build. Not even Roslyn tried to apply globbing for any of its shared projects: https://github.com/dotnet/roslyn/blob/main/src/Dependencies/Collections/Microsoft.CodeAnalysis.Collections.projitems.

Reply:
The shared project code lives in CPS, and I can tell you from experience that globbing is supported if manually editing the file, but any changes to the shared project will overwrite the glob and put back this individual list of files, so its not worth updating.

Reply:
> any changes to the shared project will overwrite the glob and put back this individual list of files, so its not worth updating.

That was exactly what I expected, since that's what the old project systems do too.